content/docs/deploying/k8s/ingress.md

```diff
@@ -193,38 +190,6 @@ ingress.pomerium.io/allow_public_unauthenticated_access: 'true'
 </td></tr>
 <tr><td>
 
-```yaml
-ingress.pomerium.io/allowed_domains: |
-  - pomerium.com
-```
-
-**Array of strings**. Users with email domains matched would be allowed.
-
-</td></tr>
-<tr><td>
-
-```yaml
-ingress.pomerium.io/allowed_users: |
-  - alice@pomerium.com
-  - bob@contractor.co
-```
```

Comment:
could you please add an example below that would serve as a replacement, similar to the example with domains?